<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1547115789614' stop='1547115815188'>
    <name>Navigate to RMT URL</name>
    <title>Navigate to RMT URL</title>
    <test-cases>
        <test-case start='1547115789615' status='failed' stop='1547115815187'>
            <name>Open Login Page</name>
            <title>Open Login Page</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>WebDriverError: invalid session id
                  (Driver info: chromedriver=2.42.591071 (0b695ff80972cc1a65a5cd643186d2ae582cd4ac),platform=Linux 4.15.0-43-generic x86_64)</message>
                <stack-trace>WebDriverError: invalid session id
                  (Driver info: chromedriver=2.42.591071 (0b695ff80972cc1a65a5cd643186d2ae582cd4ac),platform=Linux 4.15.0-43-generic x86_64)
                    at Object.checkLegacyResponse (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/error.js:546:15)
                    at parseHttpResponse (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/http.js:509:13)
                    at doSend.then.response (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/http.js:441:30)
                    at process._tickCallback (internal/process/next_tick.js:68:7)
                From: Task: WebDriver.takeScreenshot()
                    at Driver.schedule (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/webdriver.js:807:17)
                    at Driver.takeScreenshot (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/webdriver.js:1085:17)
                    at run (/usr/local/lib/node_modules/protractor/built/browser.js:59:33)
                    at ProtractorBrowser.to.(anonymous function) [as takeScreenshot] (/usr/local/lib/node_modules/protractor/built/browser.js:67:16)
                    at UserContext.&lt;anonymous> (/home/cnt-dev/Documents/Chutti-RMT/requisition-management/config.js:63:20)
                    at attempt (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4297:26)
                    at QueueRunner.run (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4217:20)
                    at runNext (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4257:20)
                    at /usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4264:13
                    at /usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4172:9</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>